utils/linear_probes.py

```diff
@@ -233,6 +233,8 @@ def main():
     stage2_path = f's{seed}_s2stage{seed_stage2}'   
     model_path = f'experiments/entAttr_d3cons_keep_ckpts_qa_cvdb_tveDefs_nEnts4000_eps20and10_bs256and256_pythia_1b_deduped_ADAFACTOR_two_stage/{stage1_path}'
 
+    # NOTE: by default, transformerlens does not support custom models. As a workaround, we modify transformerlens code, which is cursed (TODO: submit a PR to transformerlens).
```

Comment:
Are you sure there is no workaround? My transformers code for getting activations worked okay. Nobody will find this note easily

Reply:
Finding a nice workaround is not clearly worth our time

Reply:
Then make at least several notes so it would be at least visible